less_users/tests/test_event_done.py

```diff
@@ -0,0 +1,150 @@
+from datetime import date, timedelta
+
+import pytest
+
+from django.urls import reverse
+
+from freezegun import freeze_time
+
+from less_users.models import UserChallenge, Log
+
+
+@pytest.mark.django_db
+def test_view_events_unauthorised_follow(client):
+    url = reverse("event")
+    response = client.post(url, follow=True)
+
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
+# @pytest.mark.skip
+def test_view_events_unauthorised(client):
+    url = reverse("event")
+    response = client.post(url)
+
+    assert response.status_code == 302
+    assert response.url == "/my_challenges/"
+    # przy korzystaniu z view events użytkownik musi być zalogowany
+
+
+@pytest.mark.django_db
+def test_view_events_done_flower(client, user):
+    url = reverse("event")
+    client.force_login(user)
+    response = client.post(url, data={"done": "flower"}, follow=True)
+
+    assert response.status_code == 200
+    messages = list(response.context["messages"])
+
+    assert len(messages) == 1
+
+    message = str(messages[0])
+
+    assert message == "This challenge does not exist"
+
+
+@pytest.mark.django_db
+def test_view_events_done_challenge_pk_does_not_exist(client, user):
+    url = reverse("event")
+    client.force_login(user)
+
+    response = client.post(url, data={"done": -1}, follow=True)
+    messages = list(response.context["messages"])
+
+    assert "This challenge does not exist" == str(messages[0])
+
+
+@pytest.mark.django_db
+# @pytest.mark.skip
```

Comment:
Ekhm

Reply:
chodzi o to, że skoro już nie chcę pomijać tych testów, to mam usunąć @pytest.mark.skip, a nie zakomentować?

Reply:
Tak